build.gradle

```diff
@@ -48,48 +41,48 @@ configurations {
 dependencies {
     compileOnly("com.payline:payment-method-api:$paymentMethodApiVersion")
 
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.11.0'
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.11.0'
-    //implementation group: 'com.google.guava', name: 'guava', version: '25.0-jre'
-    implementation(group: 'com.google.http-client', name: 'google-http-client', version: '1.22.0') {
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.13.3'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.13.3'
+    implementation(group: 'com.google.http-client', name: 'google-http-client', version: '1.39.2') {
         transitive = false
     }
-    implementation(group: 'org.json', name: 'json', version: '20170516') {
+    implementation(group: 'org.json', name: 'json', version: '20201115') {
         transitive = false
     }
-    implementation(group: 'joda-time', name: 'joda-time', version: '2.8.1') {
+    implementation(group: 'joda-time', name: 'joda-time', version: '2.10.9') {
         transitive = false
     }
 
-    implementation(group: 'com.google.crypto.tink', name: 'tink', version: '1.2.1') {
+    implementation(group: 'com.google.crypto.tink', name: 'tink', version: '1.5.0') {
         transitive = false
     }
 
-    implementation('com.google.crypto.tink:apps-paymentmethodtoken:1.2.1') {
+    implementation('com.google.crypto.tink:apps-paymentmethodtoken:1.5.0') {
         transitive = false
     }
 
     testRuntimeOnly group: 'commons-lang', name: 'commons-lang', version: '2.6'
-    testImplementation group: "junit", name: "junit", version: "4.12"
-    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.21.0'
+    testImplementation group: "junit", name: "junit", version: "4.13"
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.3.0'
     testImplementation "com.payline:payment-method-api:${paymentMethodApiVersion}"
     testImplementation "com.payline:payment-method-integration:${paymentMethodIntegrationVersion}"
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.0.3'
-    testImplementation group: 'org.seleniumhq.selenium', name: 'selenium-server', version: '3.13.0'
-    testImplementation group: 'org.seleniumhq.selenium', name: 'selenium-support', version: '3.13.0'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.0'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: '5.5.0'
+    testImplementation group: 'org.seleniumhq.selenium', name: 'selenium-server', version: '3.14.0'
+    testImplementation group: 'org.seleniumhq.selenium', name: 'selenium-support', version: '3.14.0'
```

Comment:
Ne peut-on pas supprimer Selenium ?

Reply:
Done

Reply:
Dépendance Selenium utiliser dans un test IT **GooglePayIT**